Goal: Add a small man page for gwsetup. Assumes the 010 FHS patch has been applied

Status wrt upstream: Forwarded in the Geneweb ML

Index: geneweb-4.10/man/gwsetup.1
===================================================================
--- /dev/null	1970-01-01 00:00:00.000000000 +0000
+++ geneweb-4.10/man/gwsetup.1	2005-11-29 18:59:53.115564388 +0100
@@ -0,0 +1,47 @@
+.TH gwsetup 1 "2005 November 28th"
+.SH NAME
+gwsetup \- Interactive interface to Geneweb databases management
+.SH SYNOPSIS
+.B gwsetup
+[
+.I options
+]
+.br
+.SH DESCRIPTION
+.I gwsetup
+helps in
+.IR GeneWeb
+databases management.
+.PP
+The primary documentation of
+.I gwsetup
+is in the HTML documentation, which you can find in the documentation directory (/usr/share/doc/geneweb on
+Debian systems).
+.br
+Please look there for complete and up-to-date documentation.
+.PP
+.SH OPTIONS
+.PP
+.TP 8
+.B \-help
+command-line help
+.TP
+.B \-lang <string>
+default language
+.TP
+.B \-daemon
+use as a Unix daemon
+.TP
+.B \-p <port>
+select a port number (default=2316)
+.TP
+.B \-only <file>
+path to the file containing the IP address of the authorized host
+.TP
+.B \-log <file>
+redirect log trace to this file.
+.TP
+.B \-gd <dir>
+path of the geneweb share directory
+
+
